packages/shared/src/hooks/streaks/useReadingStreak.ts

```diff
@@ -50,7 +50,8 @@ export const useReadingStreak = (): UserReadingStreak => {
     queryFn: getReadingStreak,
     staleTime: StaleTime.Default,
     enabled: isLoggedIn,
-    refetchOnWindowFocus: isLoggedIn,
+    refetchOnWindowFocus: true,
```

Comment:
I think this is a bit too aggressive, it does not have to be that correct and update on post click should take care of the most common scenario of clicking "read post" and then streak updating?

Just refetching each time is not needed in my opinion.

Reply:
Interested to hear what others think.

Reply:
I tend to agree with Ante here.